lib/ansible/module_utils/network/solace/solace_utils.py

```diff
@@ -39,17 +42,35 @@
 QUEUES = 'queues'
 SUBSCRIPTIONS = 'subscriptions'
 
+""" RDP Resources """
+RDP_REST_DELIVERY_POINTS = 'restDeliveryPoints'
+RDP_REST_CONSUMERS = 'restConsumers'
+RDP_TLS_TRUSTED_COMMON_NAMES = 'tlsTrustedCommonNames'
+RDP_QUEUE_BINDINGS = 'queueBindings'
+
 """ DMR Resources """
 DMR_CLUSTERS = 'dmrClusters'
 LINKS = 'links'
 REMOTE_ADDRESSES = 'remoteAddresses'
-TLS_TRUSTED_COMMON_NAMES= 'tlsTrustedCommonNames'
+TLS_TRUSTED_COMMON_NAMES = 'tlsTrustedCommonNames'
 """ cert authority resources """
 CERT_AUTHORITIES = 'certAuthorities'
 
+################################################################################################
+# logging
+ENABLE_LOGGING = False  # False to disable
+
+
+def init_logging():
+    logging.basicConfig(filename='ansible_solace.log',
+                        level=logging.DEBUG,
+                        format='%(asctime)s - %(name)s - %(levelname)s - %(funcName)s(): %(message)s')
+    logging.info('Module start #############################################################################################')
+    return
 
 
-MAX_REQUEST_ITEMS = 1000  # 1000 seems to be hardcoded maximum
```

Comment:
Did you guys not see issues when multiple items were configured? I think we want to always request the 1000 items (unless this limitation has been removed on the broker side?)